### PR TITLE
Auth

### LINKED ISSUE #4: Auth
Authentication for users who are adding a new stolen car so that they can receive notifications when car is spotted

### PR DESCRIPTION
**Pull request by vasekstebra**
Fix #4

### CODE REVIEW

File: app/src/main/java/cz/muni/fi/a2p06/stolencardatabase/MainActivity.java [hunk 1]
```diff
@@ -20,9 +30,18 @@
     @BindView(R.id.fragment_container)
     FrameLayout mFragmentContainer;
 
+    FirebaseAuth mAuth;
```
Comment (rduriancik) [inline app/src/main/java/cz/muni/fi/a2p06/stolencardatabase/MainActivity.java]: private access modifier

File: app/src/main/java/cz/muni/fi/a2p06/stolencardatabase/MainActivity.java [hunk 2]
```diff
@@ -20,9 +30,18 @@
...
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        mAuth = FirebaseAuth.getInstance();
+        if (mAuth.getCurrentUser() == null) {
+            Intent loginIntent = new Intent(MainActivity.this, SignInActivity.class);
+            startActivity(loginIntent);
+            finish();
```
Comment (rduriancik) [inline app/src/main/java/cz/muni/fi/a2p06/stolencardatabase/MainActivity.java:42]: I'm not sure if the finish() method is called after startActivity() method. Try to click the back button after SignInActivity is launched from this activity.
Maybe you can place this if statement inside onStart() method.

File: app/src/main/java/cz/muni/fi/a2p06/stolencardatabase/SignInActivity.java [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+                                ))
+                        .build(),
+                RC_SIGN_IN
+        );
+    }
+
+    private void loginUser() {
+        Intent loginIntent = new Intent(SignInActivity.this, MainActivity.class);
+        startActivity(loginIntent);
+        finish();
```
Comment (rduriancik) [inline app/src/main/java/cz/muni/fi/a2p06/stolencardatabase/SignInActivity.java:70]: the same as in MainActivity

Comment (rduriancik) [reply app/src/main/java/cz/muni/fi/a2p06/stolencardatabase/SignInActivity.java:70]: I think the methods after startActivity are not called

File: app/src/main/java/cz/muni/fi/a2p06/stolencardatabase/MainActivity.java (hunk 2 above)
Comment (vasekstebra) [reply app/src/main/java/cz/muni/fi/a2p06/stolencardatabase/MainActivity.java:42]: I will take a look

**Pull request merged**